Fix OurWork Background

a-ys opened the pull request:
Reimplemented changes from #30 for new changes in main.

codingdn committed:
center aligned project description

codingdn committed:
resized value cards in about us for mobile view

Commit:
changed the "read more" hyperlink on Current Projects and Featured Projects to change text to blue during hover

Commit:
changed the "read more" hyperlink on Past Projects to change text to blue during hover

a-ys committed:
Fix OurWork Background

a-ys committed:
Fix OurWork Background

a-ys committed:
Format files edited in prev commit.

a-ys committed:
Format files edited in prev commit.

Commit:
Put icons in project page into projectDescription paragraph and removed
absolute css

- because that entire project description should be centered
- removed absolute positioning css because there is no reason for the
  title/description to have no width and height.
      these elements can have those properties and not having them makes zooming in and out really annoying. opted for default relative positioning because these elements can and Should have space on the page
      see https://developer.mozilla.org/en-US/docs/Web/CSS/position for
      more.

Commit:
Merge pull request #40 from Hack4Impact-UMD/alignProjectDesc

Center aligned project description

Commit:
Moved new mobile styles for height and width to default

Commit:
Merge pull request #41 from Hack4Impact-UMD/valuesCardMobile

Resized value cards in about us for mobile view

Commit:
Changed animation transition to transition property

Commit:
Merge pull request #42 from Hack4Impact-UMD/project-cards-read-more-hover-blue

Update Project cards' read more text to turn blue on hover

heylookitsme commented on frontend/src/styles/our_work/OurWorkHeader.module.css:
```diff
@@ -1,9 +1,13 @@
+}
+
+#headerContent {
+    padding-top: clamp(120px, 18vw, 224px);
```
maybe consider putting the `min-height: 989px` as with the apply page? or changing min height to be consistent

Commit:
Removed Dead Navbar from OurWork Page

- Removed because it was not displayed on ourwork page, and added
  unecessary height

Commit:
Removed Dead Navbar from OurWork Page

- Removed because it was not displayed on ourwork page, and added
  unecessary height

Commit:
Removed a comment

- was dead

Commit:
Removed a comment

- was dead

Commit:
Merge branch 'our-work-background-v2' of https://github.com/Hack4Impact-UMD/umd-website-redesign into our-work-background-v2

Commit:
Modified margins for current projects on Ourwork page

- removed unused imports for navbar and footer in ourwork page
- modified currentprojects mobile margins to look less awkward

heylookitsme reviewed (commented):
based
